src/dnnl/dnnl_graph_compiler.cpp

```diff
@@ -17,11 +17,14 @@
  */
 
 #include "dnnl_graph_compiler.h"
-#include "gc_version.h"
 #include <memory>
 #include <new>
 #include <string_view>
 
+#ifndef DNNL_HELPER_DLL_EXPORT
```

Comment:
`DNNL_HELPER_DLL_EXPORT` may not necessarily be provided by `dnnl_graph_compiler.h`, let's either explicitly include `dnnl_config.h` or better yet, define our own macros so we would depend on oneDNN less

Reply:
`dnnl_graph_compiler.h` uses `dnnl_status_t` in the function definitions. It's declared in `dnnl_common_types.h`, that also includes `dnnl_config.h`. Thus, `DNNL_HELPER_DLL_EXPORT` should always be declared in this context. I've also added the check on the line 25.

Reply:
> Thus, DNNL_HELPER_DLL_EXPORT should always be declared in this context.

It's true for now, however, there was an idea not long ago to get rid of oneDNN-related types in GC API (including `dnnl_status_t`) which will lead to that `dnnl_graph_compiler.h` will no longer include `dnnl_types.h`. So why wouldn't we secure ourselves and define the macros on our own on the GC side?